### PR TITLE
feat(evm): implement deploy space for Ethereum in editor

### PR DESCRIPTION
**Pull request by Sekhmet**
## Summary

Closes: https://github.com/snapshot-labs/sx-ui/issues/440

EVM deploy space editor (vanilla/ethTx authenticator, vanilla/compToken strategy, vanilla execution).

## Test plan

- Deploy space via form on EVM.

### CODE REVIEW

File: src/networks/evm/actions.ts [hunk 1]
```diff
@@ -29,8 +29,32 @@ export function createActions(chainId: number): NetworkActions {
...
+        proposalThreshold: bigint;
+        quorum: bigint;
+        authenticators: string[];
+        votingStrategies: string[];
+        votingStrategiesParams: string[][];
+        executionStrategies: string[];
+        metadataUri: string;
+      }
+    ) {
+      const response = await client.deploySpace({
```
Comment (bonustrack) [inline src/networks/evm/actions.ts:48]: We should force user to switch to the right network here

**Comment by bonustrack** [comment]
I tested it works but on the subgraph the metadata aren't loaded from IPFS

**Comment by Sekhmet** [comment]
> I tested it works but on the subgraph the metadata aren't loaded from IPFS

In the currently deployed contracts metadataUri isn't used on deploy. It was added recently and those contracts are not deployed yet:
https://github.com/snapshot-labs/sx-evm/commit/6184313d7aa9ae938d1a54b8545e8cfb2f363a41

**Comment by bonustrack** [comment]
@Sekhmet can you do this also: https://github.com/snapshot-labs/sx-ui/pull/444#discussion_r1116275813 ?

**Review by bonustrack — APPROVED** [review]
utACK

**Pull request merged**